### DISCUSSION TITLE
✏️ Termo: Capture the Flag (CtF)

### DISCUSSION THREAD

**Discussion by contraexemplo** [infuse-l10n-pt-br/Infuse#6] [Glossário] [answered]
### Definição
Exercício em que participantes tentam encontrar "bandeiras" (geralmente representadas por sequências de caracteres, como `BANDEIRA13EDASTD`) escondidos em programas, sites, ou arquivos vulneráveis.

**Comment by celsobessa** [comment, marked as answer]
Apesar de outra discussão eu ter defendido traduzir o máximo, para mim aqui tem uma nuance: o nível de uso de CTF (a abreviação, não o termo completo) é muito grande e traduzir talvez seja adicinar fricção demais. Talvez na primeira ocorrência num texto utilizar algo na linha de "Captura da Bandeira (ou CTF, do inglês _Capture the Flag_)" ou "CTF (do inglês Capture the Flag, para Capturar a Bandeira" e utilizar CTF no restante do texto?

**Reply by contraexemplo** [reply]
Essa é uma boa opção. Como eu comentei em #5, nós sempre podemos expandir um pouco a definição no glossário se necessário!
